src/scss/trumps/_utilities.scss

```diff
@@ -109,7 +114,7 @@
 }
 
 .u-textPad {
-    padding-left: spacing(x4);
+    padding-left: spacing(x4) !important;
```

Comment:
I wonder if we could include our utils last in the imports so we don't have to use `!important`'s 🤔
future talking point - not an issue for this PR.

Reply:
yeah is this !important really required?

Reply:
Actually should we really be using importants in any padding. if so then we're probably constructing our components wrong and it would be better to extend the class within the component i.e.

```scss
.my-class {
  @extend .u-textPad
}
```

Reply:
The issue is that when you have optional imports they _must_ be imported _after_ the main Fozzie import, which means that the component styles take precedence over the trumps.

I think it's one we could come back to in the future to discuss for sure 🚜

Reply:
@oanuna `!important` in this scenario is just enforcing the utility class really.  If people are overriding the styles provided by the utility class they shouldn't be using it 😄 

Agree used anywhere else it's a symptom of badly constructed components, but it kind of re-enforces how they should be applied here.

There's a good article on `!important` [not always being bad here](https://csswizardry.com/2016/05/the-importance-of-important/) (especially the section on utility classes).